test/index.js

```diff
@@ -16,15 +16,7 @@ test('resolves using pkg.svelte', () => {
 	const { resolveId } = plugin();
 	assert.is(
 		resolveId('widget', path.resolve('test/foo/main.js')),
-		path.resolve('test/node_modules/widget/src/Widget.html')
-	);
-});
-
-test('resolves using pkg.svelte.root', () => {
```

Comment:
why was this test removed?

Reply:
See https://github.com/sveltejs/rollup-plugin-svelte/pull/142#issuecomment-712664154 above – it's discouraged/no longer supported. And with the guise of removing outdated code, chucked it